.travis.yml

```diff
@@ -3,16 +3,15 @@ node_js: 12.14.0
 services: docker
 cache:
   npm: true
-  yarn: true
 addons:
   chrome: stable
 install:
-  - yarn install
+  - npm install
```

Comment:
I guess for CI more suitable `npm ci`, because npm would be pick versions from lock-file. In 'install' mode npm  would be try to install new available versions of packages and also would be update lock-file.

More info can be found [here](https://medium.com/better-programming/npm-ci-vs-npm-install-which-should-you-use-in-your-node-js-projects-51e07cb71e26)